website/blog/2021-11-22-dbt-labs-pr-template.md

```diff
@@ -237,3 +237,19 @@ addressed, and remove any items that are not relevant to this PR.
 {% endif %}
 
 ```
```

Comment:
Should we include instructions for getting this into your dbt project as well/instead?

Reply:
@ns-jwilliams Cool, would you be able to write up some brief instructions that I can add to this PR? If you're strapped for time, you can stick to the dbt project instructions, and we can always add the GH instructions later.

Reply:
Hey @ns-jwilliams would you have time for this this week? If not, no worries. We can create a new PR for adding the instructions when you have time. The main thing I want to do in this PR is get the headers/content targeting the "github pull request template" keyword for SEO.

Reply:
hey @johnblust! I think we go ahead and get this merged and then update the instructions in a separate PR!

Reply:
Cool! just need you to approve the review and I'll get this merged :)